dist/aframe-keyboard-controls.js

```diff
@@ -967,7 +967,7 @@
 	      }});
 
 	      // Fix for nonstandard `key` values (FF36-)
-	      if ('key' in KeyboardEvent.prototype) {
+	      if (!'key' in KeyboardEvent.prototype) {
```

Comment:
Could you omit the `dist/*` changes actually? (they get built automatically when I bump the version numbers).

Thanks!